contract/vm_callback.go

```diff
@@ -613,8 +569,8 @@ func luaSendAmount(L *LState, service C.int, contractId *C.char, amount *C.char)
 	}
 
 	// send the amount to the receiver
-	if r := sendBalance(L, senderState, cs.curState, amountBig); r != nil {
-		return r
+	if err := state.SendBalance(senderState, receiverState, amountBig); err != nil {
+		return C.CString("[Contract.sendBalance] insufficient balance: " + senderState.Balance().String() + " : " + amountBig.String())
```

Comment:
this is the 3rd time this message appears. it is a signal to use a function.
probably it was done right before

Reply:
ok. i will rollback to before, but use state.Sendbalance()

Reply:
yes, a local `sendBalance()` function that calls `state.SendBalance()`